ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -205,6 +224,20 @@ public override void OnDropViewInstance(ThemedReactContext reactContext, Border
         }
 
         /// <summary>
+        /// Sets the dimensions of the view.
+        /// </summary>
+        /// <param name="view">The view.</param>
+        /// <param name="dimensions">The output buffer.</param>
+        public override void SetDimensions(Border view, Dimensions dimensions)
+        {
+            Canvas.SetLeft(view, dimensions.X);
+            Canvas.SetTop(view, dimensions.Y);
+            view.Width = dimensions.Width;
+            view.Height = dimensions.Height;
+            SetUriFromMultipleSources(view);
```

Comment:
There's another lifecycle event we could use for this (OnAfterUpdate or something)